form_designer/models.py

```diff
@@ -109,7 +109,11 @@ def compile_message(self, form_data, template=None):
         if template:
             t = get_template(template)
         elif not self.message_template:
-            t = get_template('txt/formdefinition/data_message.txt')
+            if settings.HTML_EMAILS:
```

Comment:
Sorry if I was unclear! I meant this could be

```python
        if template:
            t = get_template(template)
        elif not self.message_template:
            t = get_template(settings.EMAIL_TEMPLATE)
        else:
            t = Template(self.message_template)
```

and the default value for `EMAIL_TEMPLATE` would be `'txt/formdefinition/data_message.txt'`.

That way we can easily change the template filename for text templates too.